third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -784,6 +787,59 @@ func resourceContainerCluster() *schema.Resource {
 	}
 }
 
+// Setting 0 for guest accelerator count has the effect of removing that block of guest_accelerators from the
```

Comment:
I think we can eliminate most of this comment block. I think we can get away with explaining

* that users can set a `count` of `0` which is equivalent to omitting the block (and the context that it was needed for modules in `0.11`) 
* this means the list of accelerators stored in state will be ordered identically to the list in config, but will be missing entries where `count` was `0`